server/models/User.js

```diff
@@ -3,15 +3,6 @@ const bcrypt = require('bcrypt');
 
 const userSchema = new Schema(
   {
-    firstName: {
-      type: String,
-      required: true,
-    },
-    lastName: {
-      type: String,
-      required: true,
-  
-    },
```

Comment:
Not necessary to remove these, although we don't have a Front End  to  handle it will be nice to have them in the future. Just remove the `required: true`.